test/spec/headerCtrlSpec.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+describe('Header Controller', function() {
+  var scope, ctrl, user, $rootScope;
+
+  beforeEach(function() {
+    module(function($provide) {
+      $provide.value('User', {});
+    });
+
+    inject(function($rootScope, $controller, Shared, Groups){
+      user = specHelper.newUser();
+
+      Shared.wrap(user);
+
+      scope = $rootScope.$new();
+
+      // Load RootCtrl to ensure shared behaviors are loaded
+      $controller('RootCtrl',  {$scope: scope, User: {user: user}});
+
+      ctrl = $controller('HeaderCtrl', {$scope: scope, User: {user: user}, Groups: Groups});
+    });
+  });
+
+  it('gets next milestone', function(){
+
+    scope.user.stats.lvl = 1;
+    scope.getNextMilestone();
+    expect(scope.nextMilestone).eql({"level": '4',  text: "Pets + Mounts Unlocked"});
```

Comment:
Nope, I believe the quotes can be removed. I'll make the change and rerun the tests.